apm-agent-core/src/main/java/co/elastic/apm/logging/LoggingConfiguration.java

```diff
@@ -19,18 +19,27 @@
  */
 package co.elastic.apm.logging;
 
+import co.elastic.apm.bci.ElasticApmAgent;
 import org.slf4j.event.Level;
 import org.slf4j.impl.SimpleLogger;
 import org.stagemonitor.configuration.ConfigurationOption;
 import org.stagemonitor.configuration.ConfigurationOptionProvider;
 import org.stagemonitor.configuration.source.ConfigurationSource;
 
+import javax.annotation.Nonnull;
+import java.io.File;
+import java.net.URISyntaxException;
 import java.util.List;
 
 public class LoggingConfiguration extends ConfigurationOptionProvider {
 
-    private static final String LOGGING_CATEGORY = "Logging";
+    private static final String SYSTEM_OUT = "System.out";
     private static final String LOG_LEVEL_KEY = "logging.log_level";
+    private static final String LOG_FILE_KEY = "logging.log_file";
+    private static final String DEFAULT_LOG_FILE = SYSTEM_OUT;
+
+    private static final String LOGGING_CATEGORY = "Logging";
+    private static final String AGENT_HOME_PLACEHOLDER = "_AGENT_HOME_";
```

Comment:
This seems a bit awkward, to me, but I guess it is the best solution while still allowing relative paths from the application home.